.travis.yml

```diff
@@ -3,23 +3,40 @@ sudo: false
 notifications:
   email: false
 
+before_cache:
+  - rm -rf _build/$MIX_HOME/lib
+
+cache:
+  directories:
+    - _build/$MIX_HOME
+
 elixir:
-  - 1.6
+  - 1.8
+  - 1.7
 
 otp_release:
+  - 21.1
   - 20.3
-  - 21.0
+  - 19.3
+  - 18.3
+
+matrix:
+  exclude:
+    - elixir: 1.8
+      otp_release: 19.3
+    - elixir: 1.8
+      otp_release: 18.3
```

Comment:
If we're excluding 18.3 for all the elixir versions we support, we should just remove it from the list above